common/beerocks/tlvf/yaml/beerocks/tlvf/beerocks_message_action.yaml

```diff
@@ -87,8 +87,7 @@ eActionOp_CONTROL:
 
   ACTION_CONTROL_CLIENT_STATISTICS_11K_REQUEST: 121
   ACTION_CONTROL_CLIENT_STATISTICS_11K_RESPONSE: 122
-  ACTION_CONTROL_CLIENT_LINK_MEASUREMENT_11K_REQUEST: 123
-  ACTION_CONTROL_CLIENT_LINK_MEASUREMENTS_11K_RESPONSE: 124
```

Comment:
please provide the tlvf changes and autogenerated files in the same commit.

and please remove the added line space

Reply:
dixed